app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,190 @@
+        app:layout_constraintStart_toStartOf="@id/flow_group"
+        app:layout_constraintBottom_toBottomOf="@id/flow_group"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/button_log_out"
+        style="@style/ButtonStyle.LogOut"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/log_out"
+        app:cornerRadius="6dp"
+        app:strokeColor="@color/log_out"
+        app:strokeWidth="2dp"
+        app:layout_constraintTop_toTopOf="@id/flow_group"
+        app:layout_constraintEnd_toEndOf="@id/flow_group"
+        app:layout_constraintBottom_toBottomOf="@id/flow_group"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/view_my_contacts"
+        style="@style/ButtonStyle.Contacts"
+        android:layout_width="0dp"
+        android:layout_height="55dp"
+        android:insetTop="0dp"
+        android:insetBottom="0dp"
+        android:text="@string/view_my_contacts"
+        android:textStyle="bold"
+        app:cornerRadius="6dp"
         app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/top_view"
+        app:layout_constraintVertical_bias="0.95"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/edit_profile"
+        style="@style/ButtonStyle.EditProfile"
+        android:layout_width="0dp"
+        android:layout_height="40dp"
+        android:insetTop="0dp"
+        android:insetBottom="0dp"
+        android:text="@string/edit_profile"
+        app:cornerRadius="6dp"
+        app:layout_constraintBottom_toTopOf="@id/view_my_contacts"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/top_view"
+        app:layout_constraintVertical_bias="0.95"
+        app:strokeColor="@color/black"
+        app:strokeWidth="2dp" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/links"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:constraint_referenced_ids="imageButtonFacebook,imageButtonLinkedIn,imageButtonInstagram"
+        app:flow_firstHorizontalBias="0.5"
+        app:flow_firstHorizontalStyle="spread_inside"
+        app:flow_horizontalGap="16dp"
+        app:layout_constraintBottom_toTopOf="@id/edit_profile"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@id/top_view" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/imageButtonFacebook"
```

Comment:
Previously you were declaring ids using underscores in naming but here you are using different style. Bring the ids to the same style. Do it for all ids.

Reply:
done